packages/rum/src/boot/postStartStrategy.ts

```diff
@@ -75,42 +100,21 @@ export function createPostStartStrategy(
 
     status = RecorderStatus.Starting
 
-    runOnReadyState(configuration, 'interactive', () => {
-      if (status !== RecorderStatus.Starting) {
-        return
-      }
-
-      const deflateEncoder = getOrCreateDeflateEncoder()
-      if (!deflateEncoder) {
-        status = RecorderStatus.Stopped
-        return
-      }
-
-      ;({ stop: stopRecording } = startRecordingImpl(
-        lifeCycle,
-        configuration,
-        sessionManager,
-        viewHistory,
-        deflateEncoder
-      ))
-
-      status = RecorderStatus.Started
-    })
+    doStart().catch(monitorError)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Maybe document why we have a promise that we don't await.